.github/actions/setup-build/action.yml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2022 The StableHLO Authors.
+
+# The setup-build action gets everything needed by buildAndTest into the workspace.
+# This checks out the proper revision of the LLVM repository, installs ninja, and sets
+# up the ccache for faster re-compilation.
```

Comment:
Consider reformulating this comment to make sure it always stays in sync with the implementaton. For example, it looks like even now it's out of sync because it doesn't call out installation of lld. Maybe drop the second sentence altogether?